src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man

```diff
@@ -4869,7 +4869,7 @@
             />
         <string
             id="PS_PROVIDER.event.E_A_CREATE_SERVER_REMOTESESSION.message"
-            value="Request %1. Creating a server remote session. UserName: %2 Custome Shell Id: %3"
```

Comment:
Please open a PR in the PowerShell-Native repo for this issue.

Reply:
Should the change to the .man file in this PR be reverted?

Reply:
No need to revert.  I'm not sure if the manifest file here is still needed, but it is not the one used by the installer.